php/example_code/lambda/Readme.md

```diff
@@ -0,0 +1,98 @@
+# AWS Lambda code examples for the SDK for PHP
+
+## Overview
+
+Shows how to use the AWS SDK for PHP to create, deploy, and invoke
+AWS Lambda functions. Learn to accomplish the following tasks
+
+*Lambda lets you run code without provisioning or managing servers. Upload your code
+and Lambda takes care of everything required to run and scale your code with high
+availability.*
+
+## ⚠️ Important
+
+* Running this code might result in charges to your AWS account.
+* Running the tests might result in charges to your AWS account.
+* We recommend that you grant your code least privilege. At most, grant only the minimum permissions required to perform the task. For more information, see [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege).
+* This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+
+## Code examples
+
+### Single action
+
+* [Create a function](LambdaService.php)
+  (`CreateFunction`)
+* [Delete a function](LambdaService.php)
+  (`DeleteFunction`)
+* [Get a function](LambdaService.php)
+  (`GetFunction`)
+* [Invoke a function](LambdaService.php)
+  (`Invoke`)
+* [List functions](LambdaService.php)
+  (`ListFunctions`)
+* [Update function code](LambdaService.php)
+  (`UpdateFunctionCode`)
+* [Update function configuration](LambdaService.php)
+  (`UpdateFunctionConfiguration`)
+
+### Scenarios
+
+* [Get started with functions](scenario_getting_started_functions.py)
+
+## Run the examples
+
+### Get started with functions
+
+This interactive scenario runs at a command prompt and shows you how to use
+AWS Lambda to do the following:
+
+1. Create an AWS Identity and Access Management (IAM) role role that grants Lambda
+   permission to write to logs.
+1. Create a Lambda function and upload handler code.
+1. Invoke the function with a single parameter and get results.
+1. Update the function code and configure its Lambda environment with an environment
+   variable.
+1. Invoke the function with new parameters and get results. Display the execution
+   log that's returned from the invocation.
+1. List the functions for your account.
+1. Delete the IAM role and the Lambda function.
+
+Install dependencies by using Composer:
+
+```
+composer install
+```
+
+After your composer dependencies are installed, you can run the interactive getting started file directly by using the
+following command from the `aws-doc-sdk-examples\php\Lambda\` directory:
+
+```
+php Runner.php
+```   
+
+Alternatively, you can have the choices automatically selected by running the file as part of a PHPUnit test with the
+following:
+
+```
+..\..\vendor\bin\phpunit LambdaBasicsTests.php
+```
+
+### Prerequisites
```

Comment:
Should the Prerequisites section come before the Get started section?

Reply:
That's a good question. We do have it after Run the examples in the template.

Reply:
There's a retro item to discuss this change, so closing for now